Revise uwsgi defaults and display.

jmchilton opened the pull request:
- In https://github.com/galaxyproject/galaxy/issues/1530 we realized that buffer-size needs to be increased for Galaxy requests - so increase the default.
- Mention thunder-lock since that should frequently be enabled - https://github.com/galaxyproject/ansible-galaxy-extras/pull/195.

nsoranzo commented on lib/galaxy/webapps/config_manage.py:
```diff
@@ -106,6 +106,11 @@
+    ('thunder-lock', {
+        'desc': """It is usually a good idea to set this to ``true`` if processes is greater than 1.""",
+        'default': 'false',
+        'type': 'str',
```
These could be changed to:
```
        'default': False,
        'type': 'bool',
```
after https://github.com/galaxyproject/galaxy/pull/5747 is merged.

jmchilton committed:
Revise uwsgi defaults and display.

In https://github.com/galaxyproject/galaxy/issues/1530 we realized that buffer-size needs to be increased for Galaxy requests. Also mention thunder-lock since that should frequently be enabled - https://github.com/galaxyproject/ansible-galaxy-extras/pull/195.

jmchilton commented:
@nsoranzo Rebased on top of fixes from #5747. I also think I fixed the other issues you found.

natefoo commented:
I think this needs to be backported to 18.01? We also need to update the scaling docs.
